pkg/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/containerd/errdefs/pkg
+
+go 1.22
+
+require (
+	github.com/containerd/errdefs v0.2.0
```

Comment:
I assume then we need to update this once we tag a new version?

Reply:
Yeah, we should update once we have the split version for sure